lib/pom.xml

```diff
@@ -126,8 +126,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.1</version>
                 <configuration>
-                    <source>8</source>
-                    <target>8</target>
+                    <source>11</source>
```

Comment:
What was broken if using Java 8 for lib project?

I think you've made a similar change in your other PR that was merged into master. You could pull the latest alcor/master to get your feature branch in sync.

Reply:
If I made this change, that was a mistake. The problem is that these snippets give runtime error in IntelliJ if jdk is set to 11, which is the default in my ENV. If I have to debug someting in IntelliJ, I change these parts to place IntelliJ and before commiting I undo those changes. In order not to get into this trouble I have left marker comments so that I can quickly change the code if I have to debug.
We need to have a different way of dealing with jdk version differences such as these.

Reply:
I am sorry, this is not my PR. The deafult browser from outlook doesn't allow me to comment on reviews, so I copy the link and open it in firefox but I might have had something else copied so I ended up here.